### PR TITLE
Adding Calista's CZ1016 Project

### PR DESCRIPTION
**Pull request by calicxy**

### CODE REVIEW

File: CZ1016_Intro to Data Science/2020/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Introduction to Data Science
 
-| Group Members              | Link to Project Repo                                       |
-| -------------------------- | ---------------------------------------------------------- |
-| Bhargav, Kah Ee, Nicolette | [CatsnDogs](https://github.com/majulahsingapuri/CatsnDogs) |
+| Group Members              | Link to Project Repo                                                |
+| -------------------------- | ------------------------------------------------------------------- |
+| Bhargav, Kah Ee, Nicolette | [CatsnDogs](https://github.com/majulahsingapuri/CatsnDogs)          |
```
Comment (majulahsingapuri) [inline CZ1016_Intro to Data Science/2020/README.md:5]: I know in your commit history you added your project to the wrong file. Do try to refrain from such commits in the future that just change the layout but not add content.

**Review by majulahsingapuri — APPROVED** [review]

**Comment by majulahsingapuri** [comment]
Merging this branch for now. No further change required

File: CZ1016_Intro to Data Science/2020/README.md (hunk 1 above)
Comment (majulahsingapuri) [reply CZ1016_Intro to Data Science/2020/README.md:5]: @johnsonfoo yes i do have a git hook set up but it doesnt work for markdown table formatting for some reason. im not sure if the hook works on your system when you commit code

Comment (johnsonfoo) [reply CZ1016_Intro to Data Science/2020/README.md:5]: Started a new issue to track this request

**Pull request merged**